src/crate/theme/rtd/conf/__init__.py

```diff
@@ -59,6 +59,10 @@
     "tracking_segment_id": environ.get("TRACKING_SEGMENT_ID", ""),
     "tracking_hubspot_id": environ.get("TRACKING_HUBSPOT_ID", ""),
     "tracking_project": "",
+    # Google custom search engine
+    "google_search_api": environ.get("GOOGLE_SEARCH_API", ""),
+    "google_search_cx": environ.get("GOOGLE_SEARCH_CX", ""),
```

Comment:
For making this work, we need those two environment variables to be established on all RTD projects. We can add them with https://github.com/crate/distribute/pull/434.